src/crypto.rs

```diff
@@ -1675,6 +1687,22 @@ mod test {
         assert_ne!(hasher256.finish(), hasher512.finish());
     }
 
+    #[test]
+    fn test_public_key_from_pem_spki() {
```

Comment:
Can we update this test to be similar to `PublicKey::from_spki`? Test for all key types etc. A bit redundant but I think it'd be nice. WDYT?

Reply:
That makes sense, I'll take a look at this :+1:

Reply:
Looks good!